ert_storage/endpoints/records.py

```diff
@@ -1,10 +1,11 @@
 import json
 from enum import Enum
 from typing import Any, List, Optional
-from fastapi import APIRouter, Depends, Header, Request
+from fastapi import APIRouter, Depends, Header, Request, HTTPException, status
 from sqlalchemy.orm.exc import NoResultFound
 from ert_storage.database import Session, get_db
 from ert_storage import database_schema as ds, json_schema as js
+from starlette.status import HTTP_409_CONFLICT, HTTP_415_UNSUPPORTED_MEDIA_TYPE
```

Comment:
Use `from fastapi.status import ` instead of directly starlette. Otherwise we "have" to add Starlette as requirement in `setup.py`. (This is a social contract that we have elsewhere too)

Reply:
Yes, that was automatically added. Will fix it.